npm/pkg/dataplane/dataplane_windows.go

```diff
@@ -43,6 +47,34 @@ func (dp *DataPlane) initializeDataPlane() error {
 	return nil
 }
 
+func (dp *DataPlane) getNetworkInfo() error {
+	err := dp.setNetworkIDByName(AzureNetworkName)
```

Comment:
We can remove the if-statement and all other code outside of it because the ticker starts right away (if I'm not mistaken)

Reply:
Ticker by default does not start right away, had to change it up a bit.